Zapper-fi/studio
# feat(unipilot): Add Unipilot integration

## Description


Added [Unipilot](https://unipilot.io/) integration. (Unipilot manages your Uniswap v3 liquidity for you, keeping it in an optimal range so you can earn higher returns and save on gas.)

## Checklist

- [- ] I have followed the [Contributing Guidelines](../CONTRIBUTING.md)
- [- ] (optional) As a contributor, my Twitter handle is: https://twitter.com/iamhbhatti18

## How to test?


Edit/create .env adding ENABLED_APPS=unipilot
Run pnpm dev to spin up nest server
Visit dev endpoints below to see result of fetchers to integrate unipilot:
[http://localhost:5001/apps/unipilot/tokens?groupIds[]=pool&network=ethereum](http://localhost:5001/apps/unipilot/tokens?groupIds[]=supply&network=ethereum)
[http://localhost:5001/apps/unipilot/tokens?groupIds[]=pool&network=polygon](http://localhost:5001/apps/unipilot/tokens?groupIds[]=pool&network=polygon)
